### PR TITLE
Mobile menu

### PR DESCRIPTION
**Pull request by eamador1**
Since the last code review, the following changes were made in this project:

- [ ] Add mobile menu following figma template design

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -45,27 +46,67 @@ html,
...
+.button-list-bottom {
+  position: absolute;
+  top: 90vh;
+  background-color: #fff;
+  width: 100vw;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 10vh;
+}
```
Comment (Gambit142) [inline style.css:76]: - [ ] One of the requirements is to make your design look very identical to that of the Figma template that you chose. For the mobile menu design, the bottom part is not identical to that of the mobile menu of the Figma template that you chose. Kindly make the design identical all through. 👍 💻 

### Your Design

image

### Figma Template

image

**Review by Gambit142 — CHANGES_REQUESTED** [review]
Hi @eamador1 @WilfriedM01 , 

Good job so far!

## Highlights
- Configured `Eslint` linter ✔️ 
- All linters checks passed ✔️ 
- Create a mobile menu with a links ✔️ 
- Closing icon functions properly ✔️ 

You still need to work on some issues to go to the next project, but you are almost there!

## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

You can also consider the following:

N/A

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### Please tag me in your question so I can receive the notification. You can tag me by typing @Gambit142

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: style.css (hunk 1 above)
Comment (eamador1) [reply style.css]: @Gambit142 if you see the figma design, it looks like the bottom that I included is part of the figma design. You can also argue that it is the bottom of a phone which does not make much sense to me., but it is ok👍👍

**Comment by eamador1** [comment]
Since the last review, the following changes were made:

- [ ] The bottom of the mobile menu was changed

**Review by Felix45 — APPROVED** [review]
# Approved :green_circle: 
Hi @eamador1, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights :heavy_check_mark: 
- [x] Mobile menu works as expected :white_check_mark: 
- [x] Good UI design :white_check_mark: 
- [x] GitHub flow is used correctly :white_check_mark: 
- [x] Repository has a professional readme file :white_check_mark:    
- [x] All linter configurations are setup correctly :white_check_mark:  

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**